setup.py

```diff
@@ -39,6 +39,7 @@
     "pyyaml>=5.1.1",
     "spacy>=2.3.0",
     "mypy>=0.782",
+    "google-cloud-translate>=3.0.1",
```

Comment:
Add this in the `extras_require` field in setup.py

Reply:
set the key to be something like `language_annotator`

Reply:
Added to the `extras_require` field in setup.py with a key of `language_annotator` as suggested.

Also updated extras-requirements.txt to:
```
-e .[bot]
-e .[tensorflow]
-e .[bert]
-e .[examples]
-e .[language_annotator]

```